com/win32com/client/makepy.py

```diff
@@ -427,12 +425,8 @@ def main():
         path = os.path.dirname(outputName)
         if path != "" and not os.path.exists(path):
             os.makedirs(path)
-        if sys.version_info > (3, 0):
-            f = open(outputName, "wt", encoding="mbcs")
-        else:
-            import codecs  # not available in py3k.
+        f = open(outputName, "wt", encoding="mbcs")
```

Comment:
unrelated to this PR, but I suspect this is wrong now - when this file is being read I believe it will be assumed utf-8. Ditto the `mbcs` reference in build.py

Reply:
Seems like a simple enough change, `encoding="mbcs"` in two files. Can be done in a separate PR.